src/Core/Vault/Authorization/Permissions/NormalCipherPermissions.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+using Bit.Core.Entities;
+using Bit.Core.Models.Data.Organizations;
+using Bit.Core.Vault.Models.Data;
+
+namespace Bit.Core.Vault.Authorization.Permissions;
+
+public class NormalCipherPermissions
+{
+    public static bool CanDelete(User user, CipherDetails cipherDetails, OrganizationAbility organizationAbility)
```

Comment:
Shouldn't `OrganizationAbility` be nullable? If the cipher belongs solely to the user then there's no `OrganizationAbility` to get.